Gemfile

```diff
@@ -1,6 +1,6 @@
 source "http://rubygems.org"
 
-gem 'rails', '~> 4.1.0'
+gem 'rails', '~> 5.0.1'
```

Comment:
Why update this to require Rails 5? I think ideally this should work with both 4 and 5 but seems unrelated to the Ruby 2.3.1 fix anyways?